nucypher/blockchain/eth/clients.py

```diff
@@ -428,6 +428,9 @@ def check_for_timeout(t):
 
         return True
 
+    def parse_transaction_data(self, transaction):
+        return transaction.data
```

Comment:
I think the correct field is `data`. It's what web3py, web3js and infura mention in their docs.
EthereumTester, however, has an open issue for this very same problem: https://github.com/ethereum/eth-tester/issues/173

I think we should use `input` here, and `data` in `EthereumTesterClient`
```suggestion
        return transaction.input
```

Reply:
Maybe we should consider fixing other places where this is incorrectly used:

* `eth/multisig.py, L46, L99?, L109?`
* somewhere else?

Reply:
yeah... I added a TODO.  The way Proposals are created needs to be refactored a bit maybe, to do this correctly.